### PR TITLE
Min tx fee limit

### PR DESCRIPTION
**Pull request by iverc**

### CODE REVIEW

File: app/ante/ante.go [hunk 1]
```diff
@@ -31,46 +30,31 @@ func NewAnteHandler(ak authkeeper.AccountKeeper, bk bankkeeper.Keeper, gasConsum
...
-	loweredGasPrice := sdk.DecCoin{
+	minGasPrice := sdk.DecCoin{
 		Denom:  "rowan",
-		Amount: sdk.MustNewDecFromStr("0.00000005"),
+		Amount: sdk.MustNewDecFromStr("100000000000000000"),
 	}
-
-	if !loweredGasPrice.IsValid() {
-		return ctx, sdkerrors.Wrap(sdkerrors.ErrLogic, "unable to lower gas price")
+	if len(msgs) == 1 && (msgs[0].Type() == disptypes.MsgTypeCreateDistribution || msgs[0].Type() == disptypes.MsgTypeRunDistribution) {
```
Comment (brianosaurus) [inline app/ante/ante.go:49]: Any reason not to loop msgs looking for one that has a min gas price? Maybe the last one has a min and [0..n-1] are zero?

Comment (iverc) [reply app/ante/ante.go:49]: @brianosaurus we were checking if len(msgs) != 1 to return error cause create/run distribution should have exactly 1 msg

**Review by brianosaurus — APPROVED** [review]
:+1:

**Review by brianosaurus — APPROVED** [review]
:+1:

**Review by intl-man — DISMISSED** [review]

**Review by intl-man — APPROVED** [review]

**Pull request merged**